meetings.py

```diff
@@ -39,17 +40,23 @@ def fetch_records(report_url, params, wait=1):
         if 'error' in response.keys():
             raise ZoomApiException(response['error'])
 
-        pages.append(response)
+        [records.append(record) for record in response[listkey]]
```

Comment:
Using list comprehensions in a void context like this is frowned upon. A `for` loop is more pythonic when what you're interested in is the effect of calling something on each member of an iterable and not the resulting list itself. That said, what II think you  really want here is:

`records.extend([record for record in response[listKey]])`